app/views/hyku/api/v1/work/_work.json.jbuilder

```diff
@@ -1,104 +1,110 @@
+  #                                         "location" => nil,
+  #                                         "material_media" => nil,
+  #                                         "migration_id" => nil,
+  #                                         "official_url" => nil,
+  #                                         "organisational_unit" => nil,
+  #                                         "outcome" => nil,
+  #                                         "page_display_order_number" => nil,
+  #                                         "pagination" => nil,
+  #                                         "participant" => nil,
+  #                                         "photo_caption" => nil,
+  #                                         "photo_description" => nil,
+  #                                         "place_of_publication" => nil,
+  #                                         "project_name" => nil,
+  json.publisher work.publisher
+  #                                         "qualification_level" => nil,
+  #                                         "qualification_name" => nil,
+  #                                         "reading_level" => nil,
+  #                                         "related_exhibition" => nil,
+  #                                         "related_exhibition_date" => nil,
+  #                                         "related_exhibition_venue" => nil,
+  json.related_url work.related_url
+  json.resource_type work.resource_type
+  #                                         "review_data" => nil,
+  #                                         "rights_holder" => nil,
+  json.rights_statement work.rights_statement
+  #                                         "series_name" => nil,
+  json.source work.source
+  json.subject work.subject
+  if work.representative_presenter&.solr_document&.public?
+    json.representative_id work.representative_id
+  else
+    json.representative_id nil
+  end
+  # json.thumbnail_base64_string nil
+  if work.thumbnail_presenter&.solr_document&.public?
+    components = {
+      scheme: Rails.application.routes.default_url_options.fetch(:protocol, 'http'),
+      host: @account.cname,
+      path: work.solr_document.thumbnail_path.split('?')[0],
+      query: work.solr_document.thumbnail_path.split('?')[1]
+    }
+    json.thumbnail_url URI::Generic.build(components).to_s
+  else
+    json.thumbnail_url nil
+  end
+  json.title work.title.first
+  json.type "work"
+  #                                         "version" => nil,
+  json.visibility work.solr_document.visibility
+  #                                         "volume" => nil,
+  json.work_type work.model.model_name.to_s
+  json.workflow_status work.solr_document.workflow_state
+
+  collection_presenters = work.member_of_collection_presenters.reject { |coll| coll.is_a? Hyrax::AdminSetPresenter }
+  collections = collection_presenters.map { |collection| { uuid: collection.id, title: collection.title.first } }
+  json.collections collections
```

Comment:
Looks like these lines are new here.  Probably copied in from the `hyku_addons` override.  If you think we should keep them here (I don't see any reason why not to add them here.) then we probably need some different test setup.  

I did a little debugging and it looks like inside of `work.member_of_collection_presenters` `current_ability` is `nil` but out in the view partial it is defined.  Fixing this should solve the failing tests.